setup.py

```diff
@@ -47,6 +47,7 @@
     'docker>=4.1.0,<4.2.0',
     'Flask-BabelEx>=0.9.4',
     'pipenv==2018.11.26',
+    'python-slugify>=4.0.0,<4.1.0',
```

Comment:
minor: I am fine with looser `>=4.0.0,<5.0.0` until the library shows they can't be trusted with semantic versioning ;)

Reply:
Deal jeje